nevergrad/optimization/base.py

```diff
@@ -339,6 +397,11 @@ def tell(self, candidate: p.Parameter, loss: tp.Loss) -> None:
             # multiobjective reference is not handled :s
             # but this allows obtaining both scalar and multiobjective loss (through losses)
             callback(self, candidate, loss)
+        if not candidate.satisfies_constraints() and self.budget is not None:
+            penalty = self.penalty_(candidate)
+            assert isinstance(loss, float)
```

Comment:
this is going to explode in multiobjective cases with constraints. What should be the "penalty" in this case?